tracing-journald/src/lib.rs

```diff
@@ -86,6 +86,8 @@ pub struct Subscriber {
     field_prefix: Option<String>,
 }
 
+const JOURNALD_PATH: &str = "/run/systemd/journal/socket";
```

Comment:
Should probably resolve the lint issue here.

Reply:
Done, I think.   I'm waiting for the CI though because I don't have any Windows to test this.